spec/rails_helper.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../config/environment', __dir__)
+
+abort("The Rails environment is running in production mode!") if Rails.env.production?
+require 'rspec/rails'
+require 'coveralls'
```

Comment:
line 7 and 8 need to be at the top of this file

Reply:
Moved them to top, fixed the coveralls wear to a coveralls wear merged